cads_adaptors/adaptors/cams_solar_rad/__init__.py

```diff
@@ -0,0 +1,102 @@
+from typing import BinaryIO
+
+from cads_adaptors.adaptors.cams_solar_rad.functions import (
+    BadRequest,
+    NoData,
+    determine_result_filename,
```

Comment:
These imports should be moved to inside the retrieve method as they are causing the CI tests to fail (jinja2 not in the core environment, and this may just be the first of a few, best and easiest to put in the retrieve I think

Reply:
I moved everything that is not in the Python Standard Library inside functions/metods.